src/canvas/helper.ts

```diff
@@ -4,30 +4,36 @@ import { GradientObject } from '../graphic/Gradient';
 import { RectLike } from '../core/BoundingRect';
 import Path from '../graphic/Path';
 
+export function safeNum(num: number, defalutNum: number = 0, negative = false) {
+    // null NaN or empty string
+    if (num === null || isNaN(num) || `${num}` === '') {
```

Comment:
Seems `isNaN(num)` also includes `null` and `''` check.

Reply:
have changed